### PR TITLE
Add tests for DeleteProgramCommand

### PR DESCRIPTION
**Pull request by branzuelajohn**

### CODE REVIEW

File: src/test/java/dog/pawbook/logic/commands/DeleteProgramCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import dog.pawbook.commons.core.Messages;
+import dog.pawbook.model.Model;
+import dog.pawbook.model.ModelManager;
+import dog.pawbook.model.UserPrefs;
+import dog.pawbook.model.managedentity.Entity;
+import javafx.util.Pair;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeleteCommand}.
```
Comment (kouyk) [inline src/test/java/dog/pawbook/logic/commands/DeleteProgramCommandTest.java:25]: the name here needs updating!

**Review by kouyk — COMMENTED** [review]
Maybe do it on a filtered list as well? there was once a bug that made it possible to do so

**Comment by branzuelajohn** [comment]
> Maybe do it on a filtered list as well? there was once a bug that made it possible to do so

done!

**Review by wei-yutong — APPROVED** [review]
lgtm!

**Pull request merged**